larva/src/main/java/org/frankframework/larva/ScenarioRunner.java

```diff
@@ -81,7 +81,7 @@ public ScenarioRunner(LarvaTool larvaTool, IbisContext ibisContext, TestConfig c
 		this.multipleThreads = config.isMultiThreaded();
 
 		String blackListDirs = AppConstants.getInstance().getProperty("larva.parallel.blacklistDirs", "");
-		parallelBlacklistDirs = new ArrayList<>(StringUtil.split(blackListDirs, ","));
+		parallelBlacklistDirs = StringUtil.split(blackListDirs);
```

Comment:
The result is already a list, don't need to make a new list with it. And don't need to specify the default delimiter.